README.md

```diff
@@ -51,7 +51,7 @@ This repo contains 3 main folders:
 *Cohort 4 (started 2020-10-05) Mondays, 17:00 ET*
 
 - 2020-10-05 Chapter 1 - Introduction: Emily Wissel
```

Comment:
We tend to organise these as only "upcoming" meetings - so if those meetings have already happened they should be removed from the list!

Reply:
lmao i see you on this already